flow-typed/npm/jest_v25.x.x.js

```diff
@@ -540,6 +540,12 @@ interface JestExpectType {
     JestJQueryMatchersType &
     JestStyledComponentsMatchersType &
     JestExtendedMatchersType;
+
+  // Custom Kiwi.com expect:
```

Comment:
Is this from the actual flow-typed library, or did you add it?

Reply:
I copied this from deleted files, I will change the comment at least 😅